cms/envs/aws.py

```diff
@@ -87,6 +87,20 @@
     DEFAULT_PRIORITY_QUEUE: {}
 }
 
+# Setup alternate queues, to allow access to cross-process workers
+ALTERNATE_QUEUE_ENVS = os.environ.get('ALTERNATE_SERVICE_VARIANTS', '').split()
```

Comment:
If everyone is cool with this approach, I'll also need to make sure we get a configuration PR made to set this variable on prod/stage